### PR TITLE
Alternative way to find handler to call

### PR DESCRIPTION
**Pull request by David-Desmaisons**
In this fork, I added an alternative way to find handler that is a mix between alternative 1 and 3.
To use it,  you need to create a binding on element that should handle event and a atribute on the child element that receive the event.
In this way it has the power of solution 3 where you can use function and not a method name, and it only add a binding on element that holds the handler which is often the father element and not the child, thus with less overhead than solution 3.
The functionally is explained in the readme documentation, tested in spec document and I also added an example2.html to ilustrate its usage.
Finally, this fork is also 100% compatible with orginal version as original way to find handler are still working (original tests are untouched and passing).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,49 @@ There are a couple of cases where this technique might be necessary:
...
+
+Here is an example how you can use the `delegatedFatherClick` binding to associate multiple elements with diferent functions.
+
+```html
+<ul data-bind="delegatedHandler: 'click', foreach: items, delegatedFatherClick:{remove:removeItem,add:AddItem}">
+    <li>
+        <span data-bind="text: name"></span>
+        <a href="#" data-click-father="removeItem">x</a>
+        <a href="#" data-click-father="AddItem">+/a>
+    </li>
```
Comment (IanYates) [inline README.md]: Assuming the data-click-father attributes key into the object of the delegatedFatherClick binding, should these be

`data-click-father="remove"`   (which would then call the `removeItem` function)
and similar for the add.

**Comment by IanYates** [comment]
Would data-click-parent be a better name?

**Comment by IanYates** [comment]
I should add though, that this sounds like a great idea :)  I like the idea of being able to do the add/remove in the one binding and the 2nd example (possibly with the function names fixed if I understand correctly) seems quite straightforward to use.

File: README.md (hunk 1 above)
Comment (David-Desmaisons) [inline README.md]: You are correct Ian. I will change the documentation.

**Comment by David-Desmaisons** [comment]
Ian, I fix the documentation example and update the naming from father to parent as you suggested.

**Comment by IanYates** [comment]
:+1:

**Comment by rniemeyer** [comment]
This functionality looks interesting/useful/good to me. As there is quite a bit of code, it will just take me some time to review it thoroughly to merge. I do like the name change from `father` to `parent`. Thanks!

**Comment by rniemeyer** [comment]
@David-Desmaisons - I haven't forgot about this one. I am still interested in getting this merged in, but haven't had time to review in detail yet. Thanks!

**Comment by David-Desmaisons** [comment]
Hi @rniemeyer, any update on this? I want to use your lib as a bower dependency, and I will be perfect if it contained the new funcionality. Thanks.

**Comment by rniemeyer** [comment]
Sorry for the delay. I will move it to the top of my list and merge it in the next day or two

**Comment by David-Desmaisons** [comment]
image

**Comment by rniemeyer** [comment]
@David-Desmaisons - I am working on the merge. Didn't quite get through it yet.

**Comment by rniemeyer** [comment]
@David-Desmaisons - Thanks much for your contribution! It has been rebased and merged with some additional, mostly stylistic tweaks.

**Comment by David-Desmaisons** [comment]
Thanks!

**Pull request closed without merging**